ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productcart/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:description="Product Cart"
+    jcr:primaryType="cq:Component"
+    jcr:title="productCart"
```

Comment:
@nitheshkumar-hcl : title should not be camel case , put as : Product Cart

Reply:
Changed the title